### DISCUSSION TITLE
Multiple Instances / Stations

### DISCUSSION THREAD

**Discussion by aequireg** [neilenns/ambientweather2mqtt#164] [Q&A]
Has anyone found a clean way to support more than one station?
I installed separate instances for the units but the broker is seeing clientid collisions as it's set to "ambientweather2mqtt".

**Comment by neilenns** [comment]
You shouldn't have to install separate instances. Just one is fine, point all weather stations to it.

**Reply by aequireg** [reply]
The .env has a station MAC address input, and it doesn't seem to work for more than one in an array.

**Reply by neilenns** [reply]
Ugh, you're right, forgot about that. Hmmmmmm. I wonder why I even did this since the mac address comes in the post message from the station. I'll see if I can dig into it a bit this weekend.

**Comment by petep0p** [comment]
so, this isn't really a solution for this add-on, but i have 2 stations supported by using 2 different add-ons. i use this addon for one and the ecowitt2mqtt (which has an ambient weather mode) for the other. works great. both are really great add-ons, but expose different entities.
